pkg/management/authorizationutil.go

```diff
@@ -53,3 +54,42 @@ func checkTarget(authorizations []configuration.Authorization, allowedTargetReal
 	}
 	return nil
 }
+
+// Validate the scope of each authorization in the array. Returns an error if an authorization is not valid
+func ValidateScope(authorizations []configuration.Authorization) error {
```

Comment:
Rename to validateScope(s) (it is not exposed in other packages)
Do we need to have a slice of Authorization (`validateScopes`) ? or just one Authorization (`validateScope`) ?

Reply:
Renamed in commit 4961ed1d33150cddad65dbaaa7db1b6eddc37eff 

In one call I need to validate a slice in another call only one Authorization. I choosed the slice to have a similar signature than the method Validate from the same file.

Reply:
Ok... I didn't realized you had two calls to this function. So you could have 2 functions: validateScope and validateScopes... the second one calling the first one.

Reply:
Yes, it is cleaner indeed. Corrected in commit 232ffd8a3beb0736a72bdcb2c13507fd55cf59f7